src/room.ts

```diff
@@ -354,28 +400,49 @@ export class Room extends EventEmitter implements Sayable {
 
     return null
   }
-
   /**
-   * NickName / DisplayName / RemarkName of member
+   * find member by `nick`(NickName) / `display`(DisplayName) / `remark`(RemarkName)
+   * when use member(name:string), find member by nickName by default
    */
-  public member(name: string): Contact | null {
-    log.verbose('Room', 'member(%s)', name)
+  public member(queryArg: MemberQueryFilter | string): Contact | null {
+    let query: MemberQueryFilter
+    if (typeof queryArg === 'string') {
```

Comment:
Need to support `member(name: string)` here, which name could be any of remark/display/nick.